src/UI/SignTransactionsModals/SignWithDeviceModal/SignStep.tsx

```diff
@@ -89,9 +90,14 @@ const SignStepComponent = (props: SignStepType & WithStylesImportType) => {
 
   const onSubmit = async () => {
     await onSignTransaction();
+
     if (signLastTransaction && GuardianScreen) {
       return setShowGuardianScreen(true);
     }
+
+    if (signLastTransaction && handleSubmit) {
```

Comment:
?

Reply:
Useful if we want to do an action on submit